src/sas/qtgui/Perspectives/Invariant/InvariantPerspective.py

```diff
@@ -412,13 +414,15 @@ def calculateThread(self, extrapolation):
             reactor.callFromThread(self.updateModelFromThread, WIDGETS.W_SPECIFIC_SURFACE_ERR,
                                    surface_error)
 
-        qstar_total += qstar_low + qstar_high
+        qstar_total = qstar_data + qstar_low + qstar_high
         qstar_total_error = np.sqrt(
-            qstar_total_error * qstar_total_error
+            qstar_data_err * qstar_total_error
```

Comment:
Shouldn't be "qstar_data_err * qstar_data_err" ?

Reply:
Hmm. Will need @butlerpd to check this one. Where these things are calculated and passed from is a bit of a can of worms as you no doubt discovered.

Reply:
Thanks @smk78 for letting me take a look - sometimes weird things happen in this invariant code indeed.... but in this case I am thoroughly embarrassed. @wpotrzebowski is correct. Here we are just adding the errors in quadrature not doine sophisticated error propagation through a function (where something like this might be used).  My bad (must have been in a hurry to get to something else ☹️  really good catch on the review!